atomic_reactor/util.py

```diff
@@ -542,7 +542,7 @@ def print_version_of_tools():
     ('release', 'Release'),
     ('architecture', 'Architecture'),
     ('vendor', 'Vendor'),
-    ('authoritative-source', 'Authoritative_Registry'),
+    ('authoritative-source-url', 'Authoritative_Registry'),
```

Comment:
Is this actually used in the code? If not maybe it should be removed altogether.